src/ROOTFrameReader.cc

```diff
@@ -201,7 +202,11 @@ void ROOTFrameReader::openFiles(const std::vector<std::string>& filenames) {
   // NOTE: We simply assume that the meta data doesn't change throughout the
   // chain! This essentially boils down to the assumption that all files that
   // are read this way were written with the same settings.
```

Comment:
Could you add a sentence here, that adding to the chain is essentially only done for checking the existence of the files and not really for a consistency check.